backend/de.metas.async/src/main/java/de/metas/async/api/impl/WorkPackageQuery.java

```diff
@@ -131,15 +136,15 @@ public void setSkippedTimeoutMillis(long skippedTimeoutMillis)
 	/**
 	 * @param packageProcessorIds the packageProcessorIds to set
 	 */
-	public void setPackageProcessorIds(List<Integer> packageProcessorIds)
+	public void setPackageProcessorIds(final List<Integer> packageProcessorIds)
 	{
 		if (packageProcessorIds == null)
 		{
 			this.packageProcessorIds = null;
 		}
 		else
 		{
-			this.packageProcessorIds = new ArrayList<Integer>(packageProcessorIds);
+			this.packageProcessorIds = new ArrayList<>(packageProcessorIds);
```

Comment:
consider using ImmutableSet.copyOf.
also, instead of Integer u might consider using the ID object.